spec/models/group_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe Group, type: :model do
+  # pending "add some examples to (or delete) #{__FILE__}"
+end
```

Comment:
- [ ] I noticed you've not written any test, and it is required to write test, not only because of this project but in every code we write so as to test the result of the code we wrote so I will encourage you to write unit tests for all the most important components of the RoR application and also integration tests for all the most important components of the RoR application.